src/Abc.Zebus/Hosting/HostInitializer.cs

```diff
@@ -14,6 +14,8 @@ public abstract class HostInitializer
         /// </summary>
         public virtual int Priority => 0;
 
+        public virtual bool CallAfterStartBeforeMessageDeliveryHasStarted => false;
```

Comment:
Wouldn't a different function like `AfterStartBeforeDispatch` be better?

Reply:
I considered it but thought that it didn't really make sense to be able to override both at the same time and therefore we shouldn't give the user the option to do so.  It also would make it easier to change the default behaviour for all our hosts at once if we are confident that the new time is better in all situations.

Reply:
@ocoanet ?